src/SixLabors.Fonts/Tables/Cff/CffDataDicEntry.cs

```diff
@@ -26,13 +26,13 @@ public override string ToString()
         {
             if (i > 0)
             {
-                builder.Append(" ");
+                builder.Append(' ');
```

Comment:
This should be slightly more performant but please try to keep changes relevant.

Reply:
I'll rollback this change, thanks!

Reply:
Oh, no sorry, I don't need you to do that. It's a good change, just surprising 😄